templates/django/backend/users/tests/test_commands.py

```diff
@@ -0,0 +1,75 @@
+import os
+import sys
+from io import StringIO
+
+import pytest
+from django.core.management import call_command
+from django.test import override_settings
+
+
+@override_settings(DEBUG=True)
+@pytest.mark.django_db
+def test_create_test_data_command_returns_success_message_when_debug_true(settings, db):
+    out = StringIO()
+    sys.stdout = out
+    call_command(
+        "create_test_data",
+        "5",
+    )
+    assert out.getvalue() == "Creating test data\n"
+
+
+@override_settings(DEBUG=False)
+@pytest.mark.django_db
+def test_create_test_data_returns_warning_message_when_debug_false(settings, db):
+    out = StringIO()
+    sys.stdout = out
+    call_command(
+        "create_test_data",
+        "1",
+    )
+    assert (
+        out.getvalue() == "!!! Generating test data is not possible, "
+        "because DEBUG is set to False which may indicate production environment!!!\n"
+    )
+
+
+@override_settings(DEBUG=True)
+@pytest.mark.django_db
+def test_delete_test_data_command_returns_success_message_when_debug_true(
+    settings, generated_data_with_custom_command, db
```

Comment:
Throws error claiming that fixture 'generated_data_with_custom_command' is not found. 
This is BTW true, because conftest.py is empty.

Reply:
all tests should pass now